docs/components/data-tables/README.md

```diff
@@ -624,6 +635,40 @@ The below example uses:
 </cdr-data-table>
 ```
 
+### Responsive Setup
+
+**CdrDataTable** relies on the `mounted` lifecycle hook to set up some responsive functionality. Use `v-if` to ensure that **CdrDataTable** doesn't render before the data is available.
+
+```vue
+<template>
+  <cdr-data-table
+    :col-headers="colHeaders"
+    :row-headers="rowHeaders"
+    :row-data="rowData"
+    :key-order="keyOrder"
+    v-if="hasData"
+  />
```

Comment:
To match consistency and clarity of the doc site.  I recommend using the full closing tag.  So `</cdr-data-table>` in this instance.

Reply:
Can I push back on this? We have several instances of examples using self-closing tags for this component as well as others. Furthermore, using a self-closing tag here is showing that slots are not being used. The data-table docs are specifically talking about this difference.

Reply:
Fair enough.